### PR TITLE
Update grammar of useDefaultCookieConsent setting

### PR DESCRIPTION
**Pull request by pt1602**
### 1. Why is this change necessary?

To fix the current existing typo in the german help text of _useDefaultCookieConsent_ setting in the basic information.

### 2. What does this change do, exactly?

Fixes a typo in the german help text of the _useDefaultCookieConsent_ setting in the basic information.

### 3. Describe each step to reproduce the issue or behaviour.

Open the administration in german and navigate to  "Einstellungen > Stammdaten". Scroll down to the "Sicherheit und Datenschutz" section. Now hover over the question mark right next to the "Standart Cookiehinweis verwenden" switch field. You should see the following text:

```text
Sie können hier den Standart Cookie Hinweis deactivieren wenn Sie eine Lösung eines Drittanbieters nutzen.
```

__deactivieren__ should be replaced with __deaktivieren__

image

### 4. Please link to the relevant issues (if any).

…

### 5. Checklist

- [x] I have rebased my changes to remove merge conflicts
- [ ] I have written tests and verified that they fail without my change
- [x] I have created a [changelog file](https://github.com/shopware/platform/blob/trunk/adr/workflow/2020-08-03-implement-New-Changelog.md) with all necessary information about my changes
- [ ] I have written or adjusted the documentation according to my changes
- [ ] This change has comments for package types, values, functions, and non-obvious lines of code
- [x] I have read the contribution requirements and fulfil them.

### CODE REVIEW

File: src/Core/System/Resources/config/basicInformation.xml [hunk 1]
```diff
@@ -152,7 +152,7 @@
             <label>Use Default Cookie Notification</label>
             <label lang="de-DE">Standard Cookiehinweis verwenden</label>
```
Comment (tinect) [inline src/Core/System/Resources/config/basicInformation.xml]:
```suggestion
            <label lang="de-DE">Standard-Cookiehinweis verwenden</label>
```

File: src/Core/System/Resources/config/basicInformation.xml [hunk 2]
```diff
@@ -152,7 +152,7 @@
...
             <helpText>You can deactivate the default cookie notification if you are using a third party solution.</helpText>
-            <helpText lang="de-DE">Sie können hier den Standard Cookie Hinweis deactivieren wenn Sie eine Lösung eines Drittanbieters nutzen.</helpText>
+            <helpText lang="de-DE">Sie können hier den Standard Cookie Hinweis deaktivieren wenn Sie eine Lösung eines Drittanbieters nutzen.</helpText>
```
Comment (tinect) [inline src/Core/System/Resources/config/basicInformation.xml]:
```suggestion
            <helpText lang="de-DE">Sie können hier den Standard-Cookiehinweis deaktivieren, wenn Sie eine Lösung eines Drittanbieters nutzen.</helpText>
```

**Review by tinect — CHANGES_REQUESTED** [review]
Correct the grammar, adjust the texts.

**Comment by pt1602** [comment]
Hello @tinect ,

thank you for the recommended grammar changes. I updated my PR.

File: src/Core/System/Resources/config/basicInformation.xml [hunk 3]
```diff
@@ -150,9 +150,9 @@
             <name>useDefaultCookieConsent</name>
             <bordered>bordered</bordered>
             <label>Use Default Cookie Notification</label>
-            <label lang="de-DE">Standard Cookiehinweis verwenden</label>
+            <label lang="de-DE">Standard-Cookiehinweis verwenden</label>
```
Comment (GitEvil) [inline src/Core/System/Resources/config/basicInformation.xml]:
```suggestion
            <label lang="de-DE">Standard-Cookie-Hinweis verwenden</label>
```

File: src/Core/System/Resources/config/basicInformation.xml [hunk 4]
```diff
@@ -150,9 +150,9 @@
...
             <helpText>You can deactivate the default cookie notification if you are using a third party solution.</helpText>
-            <helpText lang="de-DE">Sie können hier den Standard Cookie Hinweis deactivieren wenn Sie eine Lösung eines Drittanbieters nutzen.</helpText>
+            <helpText lang="de-DE">Sie können hier den Standard-Cookiehinweis deaktivieren wenn Sie eine Lösung eines Drittanbieters nutzen.</helpText>
```
Comment (GitEvil) [inline src/Core/System/Resources/config/basicInformation.xml]:
```suggestion
            <helpText lang="de-DE">Du solltest den Standard-Cookie-Hinweis hier nur deaktivieren, wenn Du eine alternative Drittanbieterlösung verwendest.</helpText>
```

File: src/Core/System/Resources/config/basicInformation.xml [hunk 5]
```diff
@@ -150,9 +150,9 @@
             <name>useDefaultCookieConsent</name>
             <bordered>bordered</bordered>
             <label>Use Default Cookie Notification</label>
```
Comment (GitEvil) [inline src/Core/System/Resources/config/basicInformation.xml]:
```suggestion
            <label>Use default cookie notification</label>
```

File: src/Core/System/Resources/config/basicInformation.xml [hunk 6]
```diff
@@ -150,9 +150,9 @@
...
-            <label lang="de-DE">Standard Cookiehinweis verwenden</label>
+            <label lang="de-DE">Standard-Cookiehinweis verwenden</label>
             <helpText>You can deactivate the default cookie notification if you are using a third party solution.</helpText>
```
Comment (GitEvil) [inline src/Core/System/Resources/config/basicInformation.xml]:
```suggestion
            <helpText>You should deactivate the default cookie notification only if you are using an alternative third-party solution.</helpText>
```

**Review by GitEvil — CHANGES_REQUESTED** [review]

File: src/Core/System/Resources/config/basicInformation.xml (hunk 3 above)
Comment (tinect) [reply src/Core/System/Resources/config/basicInformation.xml]: I thought of it, but it looks weirder to me 🙈

**Comment by pt1602** [comment]
Hello @GitEvil and @tinect ,

I updated both label and help text, as well pushed the changes to the PR.

**Comment by keulinho** [comment]
Thanks again for your contribution, your PR was merged!  :tada: :blue_heart:

**Pull request closed without merging**